.github/workflows/test-github-action.yml

```diff
@@ -1,6 +1,6 @@
 name: test-github-action
 
-on: [push]
+on: [ push ]
```

Comment:
My formatter would disagree :grimacing: 
Worth leaving out? Or was this intentional?